helm/prometheus-operator-app/templates/_helpers.tpl

```diff
@@ -5,7 +5,7 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 app.kubernetes.io/version: "{{ .Chart.Version }}"
 app.kubernetes.io/part-of: {{ template "kube-prometheus-stack.name" . }}
-application.giantswarm.io/team: atlas
+application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | default "atlas" | quote }}
```

Comment:
@hervenicol this works with giantswarm/architect v4.26.0 in circle ci and validate enabled